.changeset/great-humans-return.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': patch
+---
+
+Store additional, late registered and lazy loaded APIs in the registry as well.
+Route paths, route objects and others are collected already on late registration, this adds APIs to be collected also.
+Enables late registration of plugins into the application and updates ApiHolder when additional plugins have been added in.
```

Comment:
Bit of déjà vu here 😅, feels like the same thing is repeated thrice. Perhaps condense it just a lil bit? :p

Reply:
Lol, yeah that reads terribly. Modified.